ereolen_support/ereolen_support.admin.form.inc

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * @file
+ * Add eReolen support admin form.
+ */
+
+/**
+ * Create backend admin form.
+ *
+ * @param array $form
+ *   The form to create.
+ * @param array $form_state
+ *   The state of the form.
+ *
+ * @return mixed
+ */
+function ereolen_support_config_form($form, &$form_state) {
+  $fieldTypesBuild = ['select', 'textfield', 'textarea', 'emailfield', 'date_popup'];
+  $frontendForm = _ereolen_support_retrieve_form('ereolen_support_form');
+
+  $form['support_form'] = array (
+    '#type' => 'vertical_tabs',
+  );
+  $form['jira_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Jira settings'),
+    '#group' => 'support_form',
+    '#collapsible' => FALSE,
+    '#access'=> user_access('administer ereolen support form')
+  );
+  $form['field_config'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Field config'),
+    '#group' => 'support_form',
+    '#collapsible' => FALSE,
+  );
+  $form['jira_settings']['jira_domain'] = array(
```

Comment:
Missing descriptions with maybe information about how to obtain these informations in these jira-settings (maybe also default value for those possible)?

Reply:
Added description. I don't believe a default value makes much sense.